test/01LessonOneTest.js

```diff
@@ -5,36 +5,53 @@ var assert = require('assert');
 
 function returnTrue() {
     // should return true
-
+    return true;
 }
 
 function returnFalse() {
     // should return false
+    return false;
 
 }
 
 function greaterThan(big, small) {
     // should return true if the first argument is greater than the second argument
     // and return false if the first argument is less than the second argument
-
+    if (big > small) {
```

Comment:
can you try to write only:
return big>small;

That makes the code more clean and abstract.